### PR TITLE
we need the stylesheet available before initWithStyle

### LINKED ISSUE #103: Cell height
Hi, following example code:
```ruby
module MyCellStylesheet
  def my_cell_height
    300
  end

  def my_cell(st)
    st.frame = { l: 5, t: 100, w: screen_width, h: my_cell_height }
  end
end
```
No matter what I enter as `my_cell_height` (e.g. 300 or 10), the actual height of the cell doesn't change. I've tried with different cells and different heights but nothing happens. As the redpotion generator generates the code regarding the height, I assume this is the correct (and only) place to change the height.

This is what the `log` tells me:

```
(#<UITableViewCellContentView:0...)> find(self).log

 object_id   | class                 | style_name              | frame                           |
 sv id       | superview             | subviews count          | tags                            |
 - - - - - - | - - - - - - - - - - - | - - - - - - - - - - - - | - - - - - - - - - - - - - - - - |
 4635218720  | UITableViewCellContent| content_view            | {l: 0, t: 0, w: 375, h: 43.5}   |
 4635216768  | MyCell            | 2                       |                                 |
 - - - - - - | - - - - - - - - - - - | - - - - - - - - - - - - | - - - - - - - - - - - - - - - - |
RMQ 4674630768. 1 selected. selectors: [#<UITableViewCellContentView:0x11447d320>]
=> nil

(#<UITableViewCellContentView:0...)> find(self).closest(MyCell).log

 object_id   | class                 | style_name              | frame                           |
 sv id       | superview             | subviews count          | tags                            |
 - - - - - - | - - - - - - - - - - - | - - - - - - - - - - - - | - - - - - - - - - - - - - - - - |
 4635216768  | MyCell            | my_cell             | {l: 0, t: 155, w: 375, h: 44}   |
 4519189584  | UITableViewWrapperView| 3                       |                                 |
 - - - - - - | - - - - - - - - - - - | - - - - - - - - - - - - | - - - - - - - - - - - - - - - - |
```

What am I missing? Thanks!

### PR DESCRIPTION
**Pull request by squidpunch**
* Fixes #103

### CODE REVIEW

File: lib/project/ext/ui_table_view_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class UITableViewController
+  # initWithStyle is called before viewDidLoad, if we use the stylesheet in
+  # the table_data, we need to set the stylesheet earlier.
+  alias :original_initWithStyle initWithStyle
```
Comment (jamonholmgren) [inline lib/project/ext/ui_table_view_controller.rb]: Doesn't this need to be:

```ruby
alias original_initWithStyle initWithStyle
```

Comment (squidpunch) [inline lib/project/ext/ui_table_view_controller.rb]: I had a test failing on master and this fixed it. But yeah they should probably be consistent. Didn't notice I added that first ':'

Comment (squidpunch) [inline lib/project/ext/ui_table_view_controller.rb]: I'll update and push again.

Comment (squidpunch) [inline lib/project/ext/ui_table_view_controller.rb]: @jamonholmgren thanks for the keen :eyes: - I updated to match the rest of redpotion which seems to be `alias :original :changed`

**Comment by markrickert** [comment]
Good catch!

File: lib/project/ext/ui_table_view_controller.rb (hunk 1 above)
Comment (jamonholmgren) [inline lib/project/ext/ui_table_view_controller.rb]: :+1:

**Comment by squidpunch** [comment]
strangely this passed before but now looks like there is a problem (thats why other PR's are failing) so I am going to look and see why this test fails and send another PR.

**Pull request merged**